docs/docs/bootstrapping.md

```diff
@@ -39,9 +39,9 @@ In this async mode, non-bootstrapped tables are replicated as normal by the main
 ***
 
 * a bootstrap starts with a document with `type = "bootstrap-start"`
```

Comment:
no.  I mean just refer to what's output as events.  The verbage "then documents with..." is just unclear